oc-includes/osclass/model/Search.php

```diff
@@ -1135,7 +1135,7 @@ public function getPremiums($max = 2)
                 foreach($items as $item) {
                     $mStat->increase('i_num_premium_views', $item['pk_i_id']);
                 }
-                return Item::newInstance()->extendData($items);
+                return Item::newInstance()->extendData($items,true);
```

Comment:
extendData method only accept one argument, I didn't see any change in that method.

Reply:
i saw Too no change committed to item model .. but in my branch the change exists. 
```php
public function extendData($items, $single = false) {
        if (OC_ADMIN) {
            $prefLocale = osc_current_admin_locale();
        } else {
            $prefLocale = osc_current_user_locale();
        }
        if ($single) {
            return $this->singleItemsDescription($items, $prefLocale);
        } ....
```
i will delete my  fork.  and start again